cypress/fixtures/example.json

```diff
@@ -0,0 +1,5 @@
+{
```

Comment:
I tried deleting all these cypress boilerplate/sample files and every time i run the test runner, they are added. womp womp.

Reply:
Can they be .gitignored?

Reply:
oh what a good idea! updating now.